TransbankSDK.sln

```diff
@@ -28,6 +30,10 @@ Global
 		{3575F64E-4B5D-423B-890F-C267790CD303}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{3575F64E-4B5D-423B-890F-C267790CD303}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{3575F64E-4B5D-423B-890F-C267790CD303}.Release|Any CPU.Build.0 = Release|Any CPU
+		{0073916B-62AF-4CFF-9F32-0FC6610E6F89}.Debug|Any CPU.ActiveCfg = Debug|Any CPU

```

Comment:
This lines got added because you added a new project to the solution. they are not related to the content of the PR.